java-frontend/src/main/java/org/sonar/java/se/xproc/MethodBehavior.java

```diff
@@ -37,35 +36,54 @@
 
 public class MethodBehavior {
   private final Symbol.MethodSymbol methodSymbol;
-  private final boolean varArgs;
+  private boolean varArgs;
   private final int arity;
 
   private final Set<MethodYield> yields;
   private final List<SymbolicValue> parameters;
+  private final String signature;
   private boolean complete = false;
   private boolean visited = false;
+  public boolean isStaticMethod;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make isStaticMethod a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)